.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules
 dist
 dist-ssr
 *.local
+*.env
```

Comment:
Ah I think I had `.env` in my global `.gitignore` at my root directory, so I never bother putting it here. Thanks for catching!

Reply:
Yeah, i feel you have a setup that globally taking care of this, unfortunately I don't know how to get this, but it also good to have this in the gitignore for safety sake.